locust/runners.py

```diff
@@ -71,10 +73,13 @@ def on_request_failure(request_type, name, response_time, response_length, excep
         def on_hatch_complete(user_count):
             self.state = STATE_RUNNING
             if environment.reset_stats:
-                logger.info("Resetting stats\n")
+                logger.info("Resetting stats")
```

Comment:
is this change intentional?

Reply:
Yes, I thought the `\n` was a mistake. It's intentional?

Reply:
I'll revert it anyway

Reply:
Tbh I dont know :)